internal/interface/crawler/crawler.go

```diff
@@ -34,10 +45,43 @@ func NewCrawler(db *sql.DB, articlesRepo repository.ArticleRepo) *Crawler {
 }
 
 type Configuration struct {
-	RootURL             string
+	RootURL			 string
 	DesiredArticleCount int
 }
 
+func (c *Crawler) getUnvisitedURLs() ([]string, error) {
+	rows, err := c.db.Query("SELECT URL FROM CrawlStatus WHERE Visited = false LIMIT 100;")
+	if err != nil {
+		return []string{}, err
+	}
+	defer rows.Close()
+	urls := make([]string, 0, 100)
```

Comment:
We just love magic numbers, don't we? =)

Reply:
yup